app/services/encrypted_attribute.rb

```diff
@@ -54,9 +54,9 @@ def try_decrypt(encryptor, key, cost)
   end
 
   def try_decrypt_with_all_keys(encryptor, cost)
-    encryption_keys.each do |key|
-      decrypted = try_decrypt(encryptor, key, cost) or next
-      return decrypted
+    encryption_keys.each_with_object('') do |key, string|
+      decrypted = try_decrypt(encryptor, key, cost)
+      return string << decrypted if decrypted
```

Comment:
why use the empty string? Why not just throw an exception if the end of the loop is reached?